packages/nextjs/src/edge/utils/edgeWrapperUtils.ts

```diff
@@ -0,0 +1,102 @@
+import { captureException, getCurrentHub, startTransaction } from '@sentry/core';
+import { hasTracingEnabled } from '@sentry/tracing';
+import type { Span } from '@sentry/types';
+import {
+  addExceptionMechanism,
+  baggageHeaderToDynamicSamplingContext,
+  extractTraceparentData,
+  logger,
+  objectify,
+} from '@sentry/utils';
+
+import type { EdgeRouteHandler } from '../types';
+import { flush } from './flush';
+
+/**
+ * Wraps a function on the edge runtime with error and performance monitoring.
+ */
+export function withEdgeWrapping<H extends EdgeRouteHandler>(
+  handler: H,
+  options: { spanLabel: string; spanOp: string; mechanismFunctionName: string },
```

Comment:
m: why don't we use `spanDescription` instead of `spanLabel` here? I think we should keep the span option naming consistent.

Though spans having descriptions and transactions having names is still something I feel we gotta fix somehow.

applies to the other uses as well

Reply:
Yup, agree 924850f6362b4a9fcb2bf91c3f81f48eefa1fa6f